Float Package v0.5

NicoAleman opened the pull request:
- Mahony KP overwrite added to Tune config
- ATR Strength now broken up into Uphill / Downhill Strength
- Torque Tilt Strength now broken up into Accel / Regen Strength
- Turn Tilt Mode parameter removed; now only uses Yaw-Based
- AppUI Screen updates
- Float Control App compatibility updates
- Unneeded Parameters removed to resolve issues from 256 byte limit

vedderb commented:
Is there a 256-byte limit? That sounds like a bug, the config should work up to 512 bytes.

NicoAleman commented:
@surfdado ? You know more than I do since you're the one who encountered the issue. 

The package is functional from what I can tell, but when the config was beyond 256 bytes, the GET_CUSTOM_CONFIG command gets no response, no package called. For now, we cleaned up our current config to push out an update, so we could tackle the issue another time. We had cleanup to do anyways, and removed parameters that were unnecessary for our package's purposes (with the goal of keeping things more simple for the user in mind).

surfdado commented:
It is very well possible that I'm just doing something wrong. What I had observed that as soon as the package size reached 256 bytes, I would no longer see a response coming back when sending the COMM_GET_CUSTOM_CONFIG command (my app does know how to process larger messages such as MCCONF or APPCONF). Obviously more debugging is needed to track this down, this is why we limited ourselves to 256 bytes for now.
